app/views/api/v1/registrations/_info.json.jbuilder

```diff
@@ -0,0 +1,11 @@
+json.ignore_nil!
```

Comment:
I believe we are not using the `ignore_nil!` in any other view. Maybe keeping it out would be best for now. Or maybe adding it to the remainder of the views. Also, I believe there is already a partial name `_info` , maybe rendering the create could render that partial as to keep consistent with the `show` action, and keeping it DRY. WDYT?

Reply:
@juan-apa Thank you for your feedback, I agree with you and I've sent a new commit with the suggested changes.